lib/CFRPageActions.jsm

```diff
@@ -184,6 +184,16 @@ class PageAction {
    */
   async getStrings(string, subAttribute = "") {
     if (!string.string_id) {
+      if (subAttribute) {
+        return string.attributes[subAttribute];
```

Comment:
We have a similar issue of a potential exception where a localized string doesn't have any attributes and here the `string` parameter might not have `.attributes`. It seems more likely that remote provided messages/strings could be improperly structured, so we might want to fall back somehow (ignoring the fact that popupnotification has its own stricter requirements for accesskey).